From c6a1f6a3650e8487e506080eb34a244e75e37a78 Mon Sep 17 00:00:00 2001
From: Thomas Devoogdt <thomas@devoogdt.com>
Date: Tue, 1 Apr 2025 22:56:00 +0200
Subject: [PATCH] CMakeLists.txt: only require a C compiler

Commit 05ab4dc05c9d1e1f951f5849b659e2c7e291a620
"Add some catch2 tests" dropped that C flag,
but should instead only have added a simple
enable_language call for tests only.

Signed-off-by: Thomas Devoogdt <thomas@devoogdt.com>
Upstream: https://github.com/richgel999/miniz/commit/383e551cccf41fc2ce1177712d6a37a85d936642
Signed-off-by: Bernd Kuhls <bernd@kuhls.net>
---
 CMakeLists.txt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/deps/miniz/CMakeLists.txt b/deps/miniz/CMakeLists.txt
index 8978910..55d1cde 100644
--- a/deps/miniz/CMakeLists.txt
+++ b/deps/miniz/CMakeLists.txt
@@ -7,10 +7,10 @@ if(DEFINED PROJECT_NAME)
 endif()
 
 if(CMAKE_MINOR_VERSION LESS 12)
-  project(miniz)
+  project(miniz C)
   # see issue https://gitlab.kitware.com/cmake/cmake/merge_requests/1799
 else()
-  project(miniz)
+  project(miniz C)
   set(CMAKE_C_STANDARD 90)
   set(CMAKE_VERBOSE_MAKEFILE ON)
   # set(CMAKE_C_VISIBILITY_PRESET hidden)
@@ -298,6 +298,7 @@ if(BUILD_FUZZERS)
 endif()
 
 if(BUILD_TESTS)
+	enable_language(CXX)
 	set(CMAKE_CXX_STANDARD 20)
 	set(CMAKE_CXX_STANDARD_REQUIRED YES)
 	
-- 
2.55.0

